src/modules/transaction/utils/hwManager/index.js

```diff
@@ -17,19 +17,14 @@ const signTransactionByHW = async ({ wallet, schema, chainID, transaction }) =>
     chainID,
     unsignedBytes
   );
-
-  try {
-    const signature = await getSignedTransaction(
-      wallet.hw.path,
-      wallet.metadata.accountIndex,
-      unsignedMessage
-    );
-    transaction.signatures.push(signature.signature);
-    return transaction;
-    // return updateTransactionSignatures(wallet, transaction, signature);
-  } catch (error) {
-    throw new Error(error);
-  }
+  const signature = await getSignedTransaction(
+    wallet.hw.path,
+    wallet.metadata.accountIndex,
+    unsignedMessage
+  );
+  transaction.signatures.push(signature.signature);
+  return transaction;
+  // return updateTransactionSignatures(wallet, transaction, signature);
```

Comment:
💣

Reply:
@ManuGowda can we safely delete this function? I think you left it commented out when you refactored the multi signature part.

Reply:
ooo.. the multi signature is working for wallet now, it should be same for hardware wallet, did you test registering a multisign on hw and signing it?